roles/rabbitmq/templates/etc/security/limits.d/10-rabbitmq.conf

```diff
@@ -0,0 +1,3 @@
+# {{ ansible_managed }}
+rabbitmq hard nofile 2048
+rabbitmq soft nofile 2048
```

Comment:
How do you feel about making these limits a variable? Makes it easier to change, given all 3 are the same.